README.md

```diff
@@ -39,7 +39,7 @@ Tools included:
 
 ## 🚀 Usage
 
-### Launch the CLI
+ ### 🐚 Launch the CLI
```

Comment:
One space at the beginning of the line to be removed :wink:

Reply:
Fixed !